kornia/color/gray.py

```diff
@@ -125,6 +125,9 @@ class GrayscaleToRgb(Module):
         >>> output = rgb(input)  # 2x3x4x5
     """
 
+    ONNX_DEFAULT_INPUTSHAPE: ClassVar[list[int]] = [-1, 1, -1, -1]
```

Comment:
can you document somewhere how's this used for ? probable needs to be updated in the contributors docs ?